client/blocks/plugins-scheduled-updates/schedule-form.tsx

```diff
@@ -162,17 +161,14 @@ export const ScheduleForm = ( props: Props ) => {
 					setFieldTouched( { ...fieldTouched, plugins: touched } );
 				} }
 			/>
-
-			{ isEnabled( 'plugins/multisite-scheduled-updates' ) && (
-				<>
-					<Text>{ translate( 'Step 3' ) }</Text>
-					<ScheduleFormPaths
-						paths={ healthCheckPaths }
-						borderWrapper={ false }
-						onChange={ setHealthCheckPaths }
-					/>
-				</>
-			) }
+			<>
```

Comment:
In this case we don't need the Fragment wrapper anymore.

Reply:
True, removed. Thank you!